src/end.sol

```diff
@@ -4,6 +4,7 @@
 
 // Copyright (C) 2018 Rain <rainbreak@riseup.net>
 // Copyright (C) 2018 Lev Livnev <lev@liv.nev.org.uk>
+// Copyright (C) 2020-2021 Maker Ecosystem Growth Holdings, INC.
```

Comment:
Well should still be replaced by the Dai Foundation.
So whatever was under MEGH should be moved to Dai Foundation
```suggestion
// Copyright (C) 2020-2021 Dai Foundation
```